src/Algebra/Properties/Semigroup/Divisibility.agda

```diff
@@ -25,7 +26,7 @@ open import Algebra.Properties.Magma.Divisibility magma public
 
 ∣-trans : Transitive _∣_
 ∣-trans (p , px≈y) (q , qy≈z) =
-  q ∙ p , trans (assoc q p _) (trans (∙-congˡ px≈y) qy≈z)
+  (q ∙ p) , trans (assoc q p _) (trans (∙-congˡ px≈y) qy≈z)
```

Comment:
Why did you introduce brackets here? Did something change the fixities? Or it it just for clarity?

Reply:
Good catch! While thinking about how to simplify the `import`s (because divisibility is now given a custom `record` definition, rather than delegating to `Data.Product.Base.Σ`), I thought I could remove `import`s of `Data.Product.Base._,_`... doing so then causes a parse error (corrected by adding the above parentheses!), but actually can't quite be fully realised, because *mutual* divisibility again relies on `_×_`... whence my comment about redefining *that* relation in terms of `Relation.Binary.Construct.Interior.Symmetric`...

So it really draws attention to the fact that there are *several* uses of `_,_` as a `constructor`, but somehow different ones give rise to different parses, so is itself a distinct (and tricky!) issue to want to iron out maybe?

Reply:
Can revert the parentheses back (actually: should!), but still think it's worth thinking about the points above.

Reply:
I think it may be a fixity issue: overloaded constructors are indeed fine, but I haven't checked what each individual one is declared with...